students/mkdir01/lesson02/grid_printer_final.py

```diff
@@ -0,0 +1,27 @@
+def pS(x):  # printSign
```

Comment:
pep8 style guideline: function names and var names should use snake_case convention -> `def ps(x):`

also `ps` isn't very intuitive, instead of adding a comment next to function why not go ahead and name the function with that?:
```
def print_sign(sign):
```